test/show.jl

```diff
@@ -284,11 +284,11 @@ module TestShow
     showcols(io, df)
     str = String(take!(io))
     @test str == """
-    2×2 DataFrames.DataFrame
+    2×2 $DataFrame
     │ Col # │ Name │ Eltype                           │ Missing │ Values          │
     ├───────┼──────┼──────────────────────────────────┼─────────┼─────────────────┤
     │ 1     │ Fish │ String                           │ 0       │ Suzy  …  Amir   │
-    │ 2     │ Mass │ Union{Float64, Missings.Missing} │ 1       │ 1.5  …  missing │"""
+    │ 2     │ Mass │ Union{Float64, $Missing} │ 1       │ 1.5  …  missing │"""
```

Comment:
I've just realized this probably won't work on 0.7, where the `Missing` prefix won't be present, as the other columns will be less wide than in the string. So I guess we'll need two blocks with a `VERSION` check. There's another case below.

Reply:
good catch!